app/controllers/supplejack_api/concerns/story_items_controller_metrics.rb

```diff
@@ -10,16 +10,17 @@ module StoryItemsControllerMetrics
         after_action :create_story_item_interaction, only: :create
 
         def create_story_item_interaction
-          return unless @api_response[:payload] && log_request_for_metrics?
+          return unless log_request_for_metrics?
```

Comment:
Modified to use response to log `RequestMetric` as `@api_response` is removed with the DRY code.